include/seqan3/core/bit_manipulation.hpp

```diff
@@ -78,11 +78,13 @@ constexpr bool is_power_of_two(size_t n)
  */
 constexpr size_t next_power_of_two(size_t n)
 {
-    if (n == 0) return 0;
+    if (n == 0)
+        return 0;
```

Comment:
shouldn't it return 1 for 0?

Reply:
this is an edge case and undefined behaviour, because `log(0) = - infty` and `2^(log(0)) = 2^(-infty) = 1/(2^infty) = 0`